### ISSUE TITLE
"in case" vs "if"

### ISSUE THREAD

**Issue by tahliar** [SUSE/doc-styleguide#196]
This is a grammar thing rather than a style issue, but this seemed as good a place as any to mention it.

I've noticed a few places where our docs say "in case" where they should say "if". For example "In case you do not have Internet access, this step will be skipped" (not singling anyone out, there were lots of examples and this was just the simplest one I saw).

I assume that the writer used "in case" to mean "in the case that". However, "in case" is typically used more like "to preemptively prepare for the possibility". For example: "I bought a whole crate of toilet paper in case the road floods and we get stuck in our apartment".

So for the internet example above, using "in case" makes the sentence read (to me) as "This step will always be skipped because it's possible that you might not have internet, and we'd rather just avoid it entirely instead of asking."

Whereas "If you do not have Internet access, this step will be skipped" gives the proper meaning, which is "If internet = yes, then do step. If internet = no, then skip step".

tl;dr with an even simpler example:

NO: "In case you have no internet, you can't look at cat pictures."
YES: "If you have no internet, you can't look at cat pictures."
YES: "The hotel's website says they have internet, but I need my daily fix of cat pictures so I bought extra 4G data in case they're lying."

**Comment by dariavladykina** [comment]
These insights can now be found in the [Newsletter](https://confluence.suse.com/pages/viewpage.action?pageId=1127285100), so closing the issue.
